industry_benchmarks/utils/extras/extract_results.py

```diff
@@ -42,10 +42,14 @@ def load_results(f):
     # path to deserialized results
     result = json.load(open(f, 'r'), cls=JSON_HANDLER.decoder)
     if result['estimate'] is None or result['uncertainty'] is None:
-        errmsg = (
-            f"Calculations for {f} did not finish successfully!"
-            )
-        raise ValueError(errmsg)
+        click.echo(f"Calculations for {f} did not finish successfully!")
```

Comment:
I'm thinking something like this method, but without the ValueError call and maybe just bool return, called on a loop across `all_jsons`?

Reply:
the whole script is set up for a serial walk, we can loop over all the jsons first to check but it will slow things down, but I think worth it! I will make that change

Reply:
Yeah the slowdown is a bit of a concern, I guess if it becomes order minutes we have to work out if that's too much 🤔

Reply:
I'll add tdqm to the loop so people at least can see that things are working

Reply:
it added ~1 minute for 72 json files, so I think worth it!